File: NumberTranslatorStaticClassesApi/Controllers/NumberTranslatorController.cs
using Microsoft.AspNetCore.Mvc;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using NumberTranslatorStaticClassesApi.Services;

namespace NumberTranslatorStaticClassesApi.Controllers
{
    [ApiController]
    [Route("[controller]")]
    public class NumberTranslatorController : ControllerBase
    {
        [HttpGet("GetNumber/{number}")]
        public string Get(int number)
        {
            return NumberTranslatorService.Translate(number);
        }
    }
}

File: NumberTranslatorApi/Controllers/NumberTranslatorController.fs
namespace NumberTranslatorApi.Controllers

open Microsoft.AspNetCore.Mvc
open NumberTranslatorComposer


[<ApiController>]
[<Route("[controller]")>]
type NumberTranslatorController() =
    inherit ControllerBase()

    [<HttpGet("GetNumber/{number}")>]
    member _.Get (number: int) : string =
        number |> Translate